tasks/linux.sh

```diff
@@ -0,0 +1,46 @@
+inux.sh: line 19: local: can only be used in a function
```

Comment:
This stray line breaks this task entirely.

Reply:
Ah interestingly, it only breaks it when run the orchestrator. That's quite mysterious.